src/System.Private.CoreLib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -507,17 +507,35 @@ internal enum GcRestrictedCalloutKind
         [RuntimeImport(RuntimeLibrary, "RhFindBlob")]
         internal static unsafe extern bool RhFindBlob(IntPtr hOsModule, uint blobId, byte** ppbBlob, uint* pcbBlob);
 
+#if CORERT
+        internal static uint RhGetLoadedModules(IntPtr[] resultArray)
+        {
+            // CORERT-TODO: This got broken within the last week or so. We don't care for single file.
+            if (resultArray != null)
```

Comment:
Thanks! Yes, I was expecting something like that to be the long term plan.

How about `RhGetLoadedModules`? Right now it seems like the runtime doesn't care about the list of modules loaded into the process. Should that be maintained at the FX level in the array initialized at the process startup, or should the runtime maintain it?